.github/workflows/preview-publish-ga.yml

```diff
@@ -34,9 +37,11 @@ jobs:
           path: /home/runner/work/dokka/coroutines
           retention-days: 7
 
-  serialization:
+  kotlinx-serialization:
     runs-on: ubuntu-latest
-    if: github.repository == 'Kotlin/dokka'
+    if: >  # Run this job only for pull requests from forks because members with write access have web preview
+      github.event_name != 'pull_request' || 
+      (github.event_name == pull_request && github.event.pull_request.head.repo.full_name != 'Kotlin/dokka')
```

Comment:
```suggestion
      (github.event_name == pull_request && github.repository != 'Kotlin/dokka')
```

Reply:
Addressed in https://github.com/Kotlin/dokka/pull/2951#discussion_r1160151905